app/service/corpus.py

```diff
@@ -1,14 +1,16 @@
 import logging
+from typing import Optional
 
 from sqlalchemy.orm import Session
 
+import app.clients.db.session as db_session
 from app.errors import RepositoryError, ValidationError
 from app.repository import corpus_repo
 
 _LOGGER = logging.getLogger(__name__)
 
 
-def get_corpus_org_id(db: Session, corpus_import_id: str) -> int:
+def get_corpus_org_id(corpus_import_id: str, db: Optional[Session] = None) -> int:
```

Comment:
I'm intrigued why this has changed?

Reply:
This follows the pattern we have in all the other services. Before this change, the corpus service was only used in other services where the db session was passed in. I needed to use this in my ingest router and didn't want to add a dependency to the db. So I made the db parameter optional (like in other services) and it gets initialised if it is not passed in.

Reply:
Yup... that was a good refresher for me :) thanks